fhir-persistence-schema/src/main/resources/parameter_names.properties

```diff
@@ -829,4 +829,9 @@ substance-code
 substance-code:text
 substance-definition
 trigger-description
-# End of 4.3.0
\ No newline at end of file
+# End of 4.3.0
+internal-composite_target-uri_canonical_uri
+internal-composite_source-uri_canonical_uri
+internal-composite_target-uri_canonical_version
+internal-composite_source-uri_canonical_version
+# End of 5.1.0
```

Comment:
sorry, its not very clear from above but the versions listed here are actually the spec versions.
I'm not sure whether these 4 should go in the 4.0.1 section or the 4.3.0 section, but they definitely should NOT go in a new section :-)

Reply:
if you want, feel free to update the comments from `# End of 4.3.0` to `# End of FHIR 4.3.0` to make it more clear